src/app/lib/logger.const.js

```diff
@@ -58,6 +58,7 @@ const logCodes = {
 
   // Block Types
   UNSUPPORTED_BLOCK_TYPE: 'unsupported_block_type',
+  GIST_TRANSFORMATION_FAILED: 'gist_transformation_failed',
```

Comment:
Plain and simple but can you explain what this logger does?

Reply:
These are the constants we use to query our logs. If you have a look at the logging service, you'll see we have queries that make use of these event names to track requests, failures, incomplete data, etc.

This event name in particular is used here: 
https://github.com/bbc/simorgh/pull/9286/files#diff-205fed645f37d0989a7d15a741f0d3cfc86cebf38c24be1eace5a14ebedd6121R124

Whenever the data manipulation fails, we will log the metadata ID along with this constant. We can then track failures on the logging service

We use the Winston library for logging - https://github.com/winstonjs/winston